.github/CODEOWNERS

```diff
@@ -1 +1 @@
-/appinfo/info.xml    @julien-nc
+/appinfo/info.xml    @julien-nc @kyteinsky
```

Comment:
```suggestion
/appinfo/info.xml    @kyteinsky @julien-nc
```

Reply:
> If you want to match two or more code owners with the same pattern, all the code owners must be on the same line. If the code owners are not on the same line, the pattern matches only the last mentioned code owner.

All have the same priority. Why is it limited to only info.xml though?

Reply:
This is Nextcloud specific syntax, for the security bulletin done internally by the security team